lib/elixir/lib/agent.ex

```diff
@@ -260,7 +260,6 @@ defmodule Agent do
       iex> {:ok, pid} = Agent.start_link(fn -> 42 end)
       iex> Agent.get(pid, fn state -> state end)
       42
```

Comment:
The line means they are different examples and can be run in isolation. It is important to keep it. :)

Reply:
Oh, woops! My bad 😄